acceptance/examples/allow_all.rego

```diff
@@ -0,0 +1,4 @@
+# Simplest never-failing policy
+package main
+
+allow := []
```

Comment:
Is this file used anywhere now? If not then I'd suggest deleting it.

Reply:
Update: I can see it is used, but I wonder if it needs to be.

Reply:
Yes. My understanding from discussing with zregvart is that, since there is no deny rule, this triggers a 0 warning, 0 failure, 0 success count situation whereas `happy_day.rego` does not.

Reply:
Yeah, this is to help test that `ec validate` will not succeed, `allow` is not supported by Conftest, so this tests that at least one supported rule has been evaluated.